docs/indepth/installation.rst

```diff
@@ -215,7 +215,7 @@ Using Amazon Web Services
 
```

Comment:
Flocker needs to be able to log in between nodes as root for p2p volume handoff.